app/Controllers/statsController.php

```diff
@@ -73,6 +73,9 @@ public function indexAction() {
 	 *
 	 * It displays the list of idle feed for different period. The supported
 	 * periods are:
+	 *   - last 5 years
```

Comment:
Maybe also 10? 5 is peanuts, I've got tons of those.

Reply:
And I thought whether 5 is too much or not :D My goal was to add 3
I think if someone really needs a 10 years cutoff here, they can add a new pr for it